### ISSUE TITLE
Invalidating cache responses

### ISSUE THREAD

**Issue by c0debreaker** [mgonto/restangular#1069]
Forgive me since I'm posting it here. How do I invalidate cached data/responses like invalidating it after 1 hour?

Currently, I have this code

```
RestangularConfigurer.setDefaultHttpFields({cache: true});
 ```

**Comment by abhinandankothari** [comment]
@c0debreaker did you find any method ? I am also looking for the same solution

**Comment by gauravarora** [comment]
I am looking for a solution too.

**Comment by c0debreaker** [comment]
Sorry if I just saw this now @abhinandankothari and @gauravarora 

I remember I used setTimeout which calls a function that clears the object that holds the json response. Every a new url is visited, it gets inserted to an array. I have a setTimeout than runs every 5 mins that checks this array. The array contains 2 columns, the url and the time I captured it. So if let's say it reaches 5 mins, I check every recorded time from the array against the current time. If value is still less than 60 mins, don't delete it from an array. If the value is greater than 60 mins, delete it from the array then finally delete the json response.

You'll wonder why I didn't say setInterval. It's because I wrote setTimeout to act like setInterval. Just searched setInterval vs setTimeout and you'll find out why I decided to use setTimeout.

I hope this helps. If you ask if I have a code, I'm sorry I don't have it anymore because I moved to a new job. It's actually really easy.
